### DISCUSSION TITLE
Dashboard / general customisation

### DISCUSSION THREAD

**Discussion by bjmd** [firefly-iii/firefly-iii#6209] [Firefly III] [answered]
Firstly sorry if I missed this, but I can't see any reference to this in the discussions here or in the docs. 

Is there any way to customise what you see on the dashboard? For example, rearrange or add/remove items such as adding the a widget for `/budgets/list/no-budget/all`. 

And then also, is it possible to customise the columns displayed in the transactions page? For me specifically I would like to add a column to show which budget a transaction is in and then maybe remove some others which I don't need to always see, such as the source account. 

Thanks in advance.

**Comment by JC5** [comment, marked as answer]
I'm working on a new layout where more customizations will be possible.

**Reply by bjmd** [reply]
OK great, thanks.
